src/patternfly/components/HelperText/helper-text.scss

```diff
@@ -71,6 +73,15 @@
     --#{$helper-text}__item-text--FontWeight: var(--#{$helper-text}__item-text--m-error--FontWeight);
     --#{$helper-text}__item-icon--Color: var(--#{$helper-text}__item-icon--m-error--Color);
     --#{$helper-text}--m-dynamic__item-icon--Color: var(--#{$helper-text}--m-dynamic--m-error__item-icon--Color);
+
+    transition:
+      opacity 
+        var(--#{$helper-text}__item-icon--m-error--TransitionDuration--Opacity)
+        var(--#{$helper-text}__item-icon--m-error--TransitionTimingFunction--Opacity);
+    
+    @starting-style {
+      opacity: 0;
+    }
```

Comment:
Switch to using `keyframes` because `@starting-style` isn't fully supported by Firefox